tensorflow/contrib/lite/examples/ios/simple/main.m

```diff
@@ -13,10 +13,10 @@
 // limitations under the License.
 
 #import <UIKit/UIKit.h>
+#import "AppDelegate.h"
```

Comment:
As mentioned above, I think you can kill the AppDelegate entirely.

Reply:
Please look at comment above.

Reply:
ack